packages/ui/src/contexts/SwapProvider/useEvmSwap.test.tsx

```diff
@@ -96,6 +100,9 @@ describe('contexts/SwapProvider/useEvmSwap', () => {
     jest.mocked(useFeatureFlagContext).mockReturnValue({
       isFlagEnabled: (flagName) => flagName !== FeatureGates.ONE_CLICK_SWAP,
     } as any);
+    jest.mocked(useNetworkFeeContext).mockReturnValue({
+      isGaslessOn: false,
+    } as any);
```

Comment:
Please avoid using the `any` type because it disables type checking, meaning you won't get a type error if the function signature changes.

```suggestion
    } as ReturnType<typeof useNetworkFeeContext>);
```